config/example/node.yaml

```diff
@@ -60,10 +60,10 @@ grpc:
       use_insecure_crypto: true # allow using insecure ciphers with TLS 1.2
 
 tree:
-  enabled: false
-  cache_size: 10
-  replication_worker_count: 64
-  replication_channel_capacity: 64
+  enabled: true
+  cache_size: 15
+  replication_worker_count: 32
+  replication_channel_capacity: 32
```

Comment:
The support is experimental, I don't think we should `enabled: true` in the example.

Reply:
yes, the only reason why i did so is unit tests that distinguish default and non-default values

is it ok to keep the default values in examples?

Reply:
i meant `enabled: false`

we can not test that if our example files contain default values

Reply:
Ok, I think we can make it `true`, we have `testnet`/`mainnet` examples anyway.